tools/wpt/browser.py

```diff
@@ -1131,29 +1132,34 @@ def install_webdriver_by_version(self, version, dest=None):
             edgedriver_path = os.path.join(dest, self.edgedriver_name)
         else:
             bits = "win64" if uname[4] == "x86_64" else "win32"
-            edgedriver_path = os.path.join(dest, "%s.exe" % self.edgedriver_name)
-        url = "https://msedgedriver.azureedge.net/%s/edgedriver_%s.zip" % (version, bits)
+            edgedriver_path = os.path.join(dest, f"self.edgedriver_name.exe")
```

Comment:
```suggestion
            edgedriver_path = os.path.join(dest, f"{self.edgedriver_name}.exe")
```

Suggesting this without running any code, I assume the original wouldn't work but it's an untested code path in CI.